test/com/google/enterprise/adaptor/database/DatabaseAdaptorTest.java

```diff
@@ -926,4 +926,95 @@ public void testClobColumnWithNullMetadata() throws Exception {
     expected.add("col2", "null");
     assertEquals(expected, response.getMetadata());
   }
+
+  @Test
+  public void testColumnTypeDate() throws Exception {
```

Comment:
These names don't match the earlier CLOB test names. Let's have a better scheme, like testMetadataColumns_date and testMetadataColumns_dateNull. Rename the CLOB tests as well.